.gitignore

```diff
@@ -0,0 +1,356 @@
+

```

Comment:
has that file been intended for adding

Reply:
yes

Reply:
just for ur learning skills could u please delete it and commit the change as at the end of the day I will decline that PR anyway...